api/tools/bots/index.ts

```
@@ -1,10 +1,14 @@
+import { registerJoinSpeakingTool } from "./join-speaking";
 
-export function registerBotTools(server: McpServer, baasClient: BaasClient): McpServer {
+export function registerBotTools(
+  server: McpServer,
+  baasClient: BaasClient
+): McpServer {
   // Register all bot-related tools
-  const updatedServer = registerJoinTool(server, baasClient);
-  
+  // let updatedServer = registerJoinTool(server, baasClient);
+  let updatedServer = registerJoinSpeakingTool(server);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'updatedServer' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)